native/Cargo.toml

```diff
@@ -44,6 +44,7 @@ datafusion = { default-features = false, git = "https://github.com/apache/datafu
 datafusion-functions = { git = "https://github.com/apache/datafusion.git", rev = "41.0.0-rc1", features = ["crypto_expressions"] }
 datafusion-functions-nested = { git = "https://github.com/apache/datafusion.git",  rev = "41.0.0-rc1", default-features = false }
 datafusion-expr = { git = "https://github.com/apache/datafusion.git", rev = "41.0.0-rc1", default-features = false }
+datafusion-execution = { git = "https://github.com/apache/datafusion.git", rev = "41.0.0-rc1", default-features = false }
```

Comment:
We need `datafusion-execution`?

Reply:
We use `use datafusion::execution::TaskContext`. I guess we were just pulling this in transitively before via the `datafusion` crate rather than being explicit. 

We may want to avoid bringing in the core `datafusion` crate and just depend directly on the crates that we need.